### ISSUE TITLE
Aliases not implemented in command execution

### ISSUE THREAD

**Issue by TheRetr0Doc** [The-SourceCode/Open-SourceBot#35]
**Describe the bug**
https://github.com/The-SourceCode/Open-SourceBot/blob/63d2ca63e403fa496582f6f37a2bb3ee4f6ef46b/src/handler/Handler.js#L183
https://github.com/The-SourceCode/Open-SourceBot/blob/63d2ca63e403fa496582f6f37a2bb3ee4f6ef46b/src/handler/Handler.js#L195

When running command, any specified to it aliases will not work in any way.

**To Reproduce**
1. Command !help has aliase !h
2. By executing !h you will see that nothing happens.

**Expected behavior**
It should execute same code that !help does

**Screenshots**
![image](https://user-images.githubusercontent.com/36474218/60629620-1b321880-9dff-11e9-9717-0071a80097ff.png)

**Additional context**

**Comment by Jacxk** [comment]
I think it's because its missing the aliases part here:
https://github.com/The-SourceCode/Open-SourceBot/blob/63d2ca63e403fa496582f6f37a2bb3ee4f6ef46b/src/handler/Handler.js#L183

Something like this is what it's supposed to be afaik.
```js
const cmd = this.commands.get(command.toLowerCase()) || this.aliases.get(command.toLowerCase());
```

**Comment by TheRetr0Doc** [comment]
Also, I have not seen anywhere in commands usage for `isEnabled` part in code.
https://github.com/The-SourceCode/Open-SourceBot/blob/63d2ca63e403fa496582f6f37a2bb3ee4f6ef46b/src/handler/Handler.js#L184

**Comment by Swords123456** [comment]
@TheRetr0Doc The `isEnabled` part of the code is used to enable modularity, so people can enable and disable some modules/commands/events.

**Comment by SebastiaanYN** [comment]
Fixed by #36
